File: C#/CatalogoVideos/CatalogoVideos/Domain/DTO/VideoCreateRequest.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace CatalogoVideos.Domain.DTO
{
    public class VideoCreateRequest
    {
        [Required]
        public int ReponsavelId { get; set; }
        [Required]
        public int[] CategoriaIds { get; set; }
        [Required]
        public string Titulo { get; set; }

        [Required]
        public string URL { get; set; }

        public int? Idade { get; set; }


    }
}

File: Java/catalogovideos/src/main/java/br/com/iteris/universidade/catalogovideos/domain/dto/VideoCreateRequest.java
package br.com.iteris.universidade.catalogovideos.domain.dto;

import lombok.Data;

import javax.validation.constraints.NotEmpty;
import javax.validation.constraints.NotNull;

@Data
public class VideoCreateRequest {

    @NotNull(message = "O responsável é obrigatório")
    public int idResponsavel;
    @NotNull(message = "A categoria é obrigatória")
    public int[] categoriaIds;
    @NotEmpty (message = "O título é obrigatória")
    public String titulo;
    @NotEmpty(message = "A url é obrigatória")
    public String url;
}
